Diana development

DianaA96 opened the pull request:
Week 1 Development! :)

DianaA96 committed:
Update the README.md file to describe the repo.

DianaA96 committed:
Adding my introduction paragraph to index.html

DianaA96 committed:
Cleaning original document. Adding the header of the portfolio

DianaA96 committed:
Adding main tag and including the first div for intro banner

DianaA96 committed:
adding background info

DianaA96 committed:
adding my projects

DianaA96 committed:
Fixes bad indentation, adds my profiles

DianaA96 committed:
added photo gallery

DianaA96 committed:
added hidden talents

DianaA96 committed:
added funnyTrivia

DianaA96 committed:
added disclaimer

DianaA96 committed:
fixed paths to images. Added all the media required

DianaA96 committed:
imported fonts to style document. Style applied to header.

DianaA96 committed:
style applied to intro banner

DianaA96 committed:
style applied to background section, titles and spans inside the titles tags

DianaA96 committed:
style applied to my projects section

DianaA96 committed:
style applied to my profiles section

DianaA96 committed:
fixes indentation and bad tagging in html document. Style applied to footer, trivia and hidden talents.

kmontgomery commented on portfolio/src/main/webapp/index.html:
```diff
@@ -2,18 +2,136 @@
+        </div>
+      </main>
+      <footer>
+          <p>Made with 💕 by DianaA96 @ Temascalapa, MX</p>
```
Nice touch!

kmontgomery commented on portfolio/src/main/webapp/index.html:
```diff
@@ -2,18 +2,136 @@
+            </div>
+        </div>
+
+        <div id="myProfiles">
```
Nice touch with the styling here!

kmontgomery reviewed (approved):
Really well done! Loved learning more about you

Commit:
Fixing Jetty 9! :)

Commit:
This is the first time I try to code a Java servlet

Commit:
A few changes to adapt the servlet response to my own portfolio content :)

Commit:
A few changes to the index.html to render the response in the DOM

Commit:
New function to await for the server response added.

Commit:
It seems like I added tags in the servlet response that are nos necessary when I am fetching responses with Javascript. Fixing that!

Commit:
Adding gson library to pom file

Commit:
Trying to deploy a random quote!

Commit:
Fixing some bugs

DianaA96 committed:
Changes to pom.xml to deploy t
he live server. Creating an array of Java Objects to send them using JSON format. Changes applied to the Jav
a and Javascript files to fix bugs!
